pkg/apply/applier.go

```diff
@@ -197,12 +253,12 @@ func splitInfos(infos []*resource.Info) ([]*resource.Info, []*resource.Info) {
 // cancellation or timeout will only affect how long we wait for the
 // resources to become current.
 func (a *Applier) Run(ctx context.Context) <-chan event.Event {
-	ch := make(chan event.Event)
+	eventChannel := make(chan event.Event)
```

Comment:
As mentioned earlier; it might be easier to make this channel a field of the applier, and build it in the `Initialize()`

Reply:
See the comment above. Eventually I was hoping we might be able to move away from the `Initialize` function. We are still a bit away from this, but I think that should be a long-term goal.